src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/breadcrumb.html.twig

```diff
@@ -28,25 +28,30 @@
     <nav>
       <ol class="breadcrumb">
         {% for category in currentCategoryView.breadcrumb_tree %}
+          {% set isRootCategory = (category.id_parent == 0) %}
```

Comment:
Just a suggestion but is it worth displaying the root category now that it can't be edited anyway Maybe using Home as the first element of the breadcrumb would be more intuitive for the merchant?
@MatShir what do you think?

Reply:
When we are in a multistore context, we have the page after Shop Home Category creation where we have

Root Category as breadcumb
and the 2 Home categories in the list